shell/src/main/java/alluxio/cli/bundler/command/CollectConfigCommand.java

```diff
@@ -30,6 +36,10 @@
   public static final String COMMAND_NAME = "collectConfig";
   private static final Logger LOG = LoggerFactory.getLogger(CollectConfigCommand.class);
 
+  private static final Set<String> EXCLUDED_FILES = Stream.of(
+          Constants.SITE_PROPERTIES
```

Comment:
is this to avoid copying credentials?

Reply:
Yes this avoids copying unmasked credentials from alluxio-site.properties. We now do a local "alluxio getConf" which prints the credentials masked, and collect the output.

However if the credentials are in alluxio-env.sh or elsewhere (given via ALLUXIO_JAVA_OPTS for example), I don't have a way to skip/mask them.